### PR TITLE
iPhone XR, XS, XSMax Support

### LINKED ISSUE #158: Splash screen images for iPhone XR, iPhone XS, and iPhone XSMAX
Code updated to support "Splash screen images for iPhone XR, iPhone XS and iPhone XSMAX", files attached, just expanded on the above code. iPhone XS uses iPhone X code, iPhone XR and iPhone XSMAX uses same limit of limit == 896. New to all this GitHub stuff, contributing openly.

Yes I would be interested in learning to fork and submit this in a proper PR, thanks! 

[CDVSplashScreen_h.txt](https://github.com/apache/cordova-plugin-splashscreen/files/2407633/CDVSplashScreen_h.txt)
[CDVSplashScreen_m.txt](https://github.com/apache/cordova-plugin-splashscreen/files/2407634/CDVSplashScreen_m.txt)

See pull #159

### PR DESCRIPTION
**Pull request by sdgpalm2**
### Platforms affected
iPhone XR, XS, XSMax

### What does this PR do?
Added support for iPhones XR and XSMax, iPhone XS reuses iPhone X code

### What testing has been done on this change?
Tested on local copy of apps within Xcode development simulators iPhones XR, XS, and XSMax
Additional code added for XR support provided by @Stefano1964

### Closes
Closes #158

### Checklist
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

File: src/ios/CDVSplashScreen.m [hunk 1]
```diff
@@ -175,7 +181,8 @@ - (CDV_iOSDevice) getCurrentDevice
     device.iPhone6 = (device.iPhone && limit == 667.0);
     device.iPhone6Plus = (device.iPhone && limit == 736.0);
     device.iPhoneX  = (device.iPhone && limit == 812.0);
-
+    device.iPhoneXSMax  = (device.iPhone && limitNativ == 2688.0);
```
Comment (janpio) [inline src/ios/CDVSplashScreen.m]: typo? 
should probably be `limitNative` or `limit`?
(same in line below)

Comment (janpio) [reply src/ios/CDVSplashScreen.m]: this is also the reason why tests are failing on Travis for iOS

Comment (Stefano1964) [reply src/ios/CDVSplashScreen.m]: Typo, limitNative

**Comment by tobeee** [comment]
Has this been merged in?

**Comment by janpio** [comment]
@tobeee The PR is still open. So: No.

**Comment by clicksports** [comment]
@janpio When will the PR be merged?

**Comment by janpio** [comment]
After someone reviewed it that knows Objective-C - I do not. Do you? If so, try the branch and report back if it works as expected (and describe how you tested it so others can replicated)

PS: To spam contributors/members is a good way to _not_ get something merged by them.

**Comment by ermik** [comment]
Does this address iPhone Xs showing cordova default splash as opposed to older devices?

**Comment by mrsoneji** [comment]
Bump

**Comment by jcesarmobile** [comment]
Thanks for the PR, but the plugin has been integrated into cordova-ios 6, so no more iOS work will be done here.
Also, it now uses the launch storyboards, so this shouldn't be needed anymore.

**Pull request closed without merging**